code/modules/painting/paintings_custom.dm

```diff
@@ -43,6 +50,9 @@
 	// Painting
 	var/datum/painting_utensil/p = new(user, W)
 	if (p.palette.len)
+		if(jobban_isbanned(user, "artist"))
```

Comment:
You'll want to add the jobban check to `/obj/item/mounted/frame/painting/custom/attackby()` too. Or perhaps better yet, move it to `/datum/custom_painting/interact()` so any future items that support drawing check for jobban. Same goes for the add/remove from `gallery` logic probably.